snapshot/btrfs/btrfs_test.go

```diff
@@ -216,64 +215,3 @@ func (device *testDevice) remove(t *testing.T) {
 		t.Error(err)
 	}
 }
-
-func TestGetBtrfsDevice(t *testing.T) {
```

Comment:
Can we add a new test at the new location which mounts as a loopback and tests to ensure we get the correct mount info?